vc.go

```diff
@@ -95,3 +95,25 @@ func (vc VerifiableCredential) UnmarshalCredentialSubject(target interface{}) er
 		return json.Unmarshal(asJSON, target)
 	}
 }
+
+// ContainsType returns true when a credential contains the requested type
+func (vc VerifiableCredential) ContainsType(vcType URI) bool {
```

Comment:
maybe simpeler, and shorter:`IsType`? "Contains" is superfluous, because when the type contains `VerifiableCredentialType` and `NutsCredentialType` it is both